docroot/sites/all/themes/ilr_theme/scss/components/_view_modes.scss

```diff
@@ -134,20 +134,22 @@
   }
 
   body:not(.mobilizing-against-inequality) & {//don't display as card on MAI home
-    background: white;
-    border: 2px solid #e0e0e0;
-    border-bottom-width: 6px;
+    border: 1px solid $gray-light;
+    border-radius: 6px;
     @include trailer(1.5);
     @include padding-leader(1);
+    &:hover {
+      background: white;
+      border-color: $gray-medium;
```

Comment:
John, just to clarify, I can't really see the border on my external monitor for the cards at this color. I can see them on my laptop, however. Should we be concerned about that at all?